### PR TITLE
(#1434) Replace calls to Matchers with their OO counterparts

### PR DESCRIPTION
**Pull request by baudoliver7**
For #1434 :

- Replace calls to `Matchers` for test package `com.cactoos.io`

### CODE REVIEW

**Review by andreoss — CHANGES_REQUESTED** [review]

File: src/test/java/org/cactoos/io/LoggingOutputTest.java [hunk 1]
```diff
@@ -168,13 +172,13 @@ public void logAllWriteToLargeTextFile() throws Exception {
         new Assertion<>(
             "Can't log all write and close operations to text file",
```
[the remark this answers is not in the document]
Comment (baudoliver7) [reply src/test/java/org/cactoos/io/LoggingOutputTest.java]: @andreoss Done.

**Comment by baudoliver7** [comment]
@andreoss Please, see my changes.

File: src/test/java/org/cactoos/io/LoggingInputTest.java [hunk 2]
```diff
@@ -190,10 +197,10 @@ void logResetFromLargeTextFile() throws Exception {
         input.reset();
         new Assertion<>(
             "Can't log mark and reset from text file",
```
Comment (andreoss) [inline src/test/java/org/cactoos/io/LoggingInputTest.java:199]: @baudoliver7 You missed this file

**Review by andreoss — CHANGES_REQUESTED** [review]

File: src/test/java/org/cactoos/io/LoggingInputTest.java (hunk 2 above)
Comment (baudoliver7) [reply src/test/java/org/cactoos/io/LoggingInputTest.java]: @andreoss Done.

**Comment by baudoliver7** [comment]
@andreoss Please, see my changes.

**Review by andreoss — APPROVED** [review]

**Comment by baudoliver7** [comment]
@victornoel Could you please check/merge ? @andreoss approved changes.

**Comment by victornoel** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @andreoss[/z](https://www.0crat.com/u/andreoss), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Code review was too long (8 days), architects (@victornoel) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by sereshqua** [comment]
@andreoss please make sure you will find at least 3 issues during next CR, thanks

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**